apps/sms/test/marionette/lib/views/thread.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var View = require('./view');
+
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var requireFromApp = require(appRoot + '/shared/test/integration/require_from_app');
+
+function Thread() {
+  View.apply(this, arguments);
+}
+module.exports = Thread;
+
+Thread.prototype = Object.create(View.prototype);
+
+Thread.prototype.callContact = function() {
+  this.pageAccessors.Conversation.callButton.tap();
+
+  var Dialer = requireFromApp('dialer', 'lib/dialer');
```

Comment:
as I mentioned in the main discussion we can avoid this inter-app and even inter-view dependencies